app/src/main/java/com/vero/tasky/MainActivity.kt

```diff
@@ -24,7 +24,7 @@ import dagger.hilt.android.AndroidEntryPoint
 @AndroidEntryPoint
 class MainActivity : ComponentActivity() {
 
-    private val viewModel: MainViewModel by viewModels()
+    val viewModel: MainViewModel by viewModels()
```

Comment:
Why was this made public?

Reply:
I need viewModel to collect isLoggedIn State

Reply:
Where?

Reply:
But you only use this ViewModel reference in the activity? 🤔